CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.1.0-rc.2 - November 17, 2021
+
+For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
+
+### Changelog
+#### Bug fixes and improvements
+- Trigger RouteAlternativesObserver on main thread. [#5120](https://github.com/mapbox/mapbox-navigation-android/pull/5120)
+
+### Mapbox dependencies
+This release depends, and has been tested with, the following Mapbox dependencies:
```

Comment:
I know this line has been the same for all releases and I might be wrong, but for me it seems like a preposition is missing:
```suggestion
This release depends on, and has been tested with, the following Mapbox dependencies:
```